### PR TITLE
Inline telegram script

### PR DESCRIPTION
**Pull request by Matt-Esch**
I am trying to use the telegram comment widget. The widget inserts an iframe to the parent of the script node. Because scripts are appended to the end of the page, this ends up being the body and not the comments placeholder. I had to inline the script within the comments placeholder manually like this to get the comments to show in the right place.

You might have a better idea for how to do this (maybe using the scripts partial or something like that, I'm new to this templating system), but this is the expected result I'm looking for.

### CODE REVIEW

File: layouts/partials/comment.html [hunk 1]
```diff
@@ -95,10 +95,11 @@
             {{- if $telegram.outlined -}}
                 {{- $attr = printf `%v data-outlined="1"` $attr -}}
             {{- end -}}
-            {{- dict "Source" "https://comments.app/js/widget.js?2" "Defer" true "Attr" $attr | dict "Scratch" .Scratch "Data" | partial "scratch/script.html" -}}
+            <script type="text/javascript" src="https://comments.app/js/widget.js?3" {{ with $attr }} {{ . | safeHTMLAttr }}{{ end }}></script>
```
Comment (khusika) [inline layouts/partials/comment.html:98]: I think you don't need to remove `defer` attribute. It make main page page load earlier as the script is executed when the page has finished parsing.

**Comment by OrwillT** [comment]
Angthing new about this fix?

**Comment by khusika** [comment]
> Angthing new about this fix?

You can apply this commit into your personal repo if you want.

File: layouts/partials/comment.html (hunk 1 above)
Comment (uPagge) [reply layouts/partials/comment.html:98]: > I think you don't need to remove `defer` attribute. It make main page page load earlier as the script is executed when the page has finished parsing.

Thank you very much for this fix, I see you marked in my PR, but this is not your template, the developer has abandoned this project?

**Comment by uPagge** [comment]
This change I moved to the new fork :smile:

Since the developer has unfortunately abandoned the project :disappointed:, I would like to take responsibility for its support and development in the future.

:link: -> https://github.com/uPagge/uBlogger.

I focused on adding new features, refreshed the design of the posts, fixed the bugs I found. 
There were so many changes (the ability to choose a separate design for the post, breadcrumbs, a new comment system and much more), and so much to do, that I decided to give the template a **new name uBlogger.**

[Short and basic, what I have changed](https://github.com/uPagge/uBlogger/blob/master/CHANGE.md)

**I will be glad if you look at it, you may like the updated design.**

**Pull request merged**